packages/hurl/src/cli/options.rs

```diff
@@ -105,6 +107,20 @@ pub fn app(version: &str) -> Command {
                 .help("CA certificate to verify peer against (PEM format)")
                 .num_args(1)
         )
+        .arg(
+            clap::Arg::new("clientcert_file")
+                .long("cert")
```

Comment:
We can add here the short name for this option `.short('E')` to mirror curl options